tests/test_app.py

```diff
@@ -95,4 +103,8 @@ def test_get_title_data_with_uprn_match(self, mock_data, mock_title):
         response = self.app.get('/title_search_postcode/PL9_8TB')
         response_json = json.loads(response.data.decode())
         titles = response_json['titles']
+        page_number = response_json['page_number']
+        number_pages = response_json['number_pages']
         assert titles[0]['data'] == 'data'
+        assert page_number == 1
+        assert number_pages == 1
```

Comment:
It would be worth adding a test that would check if the application returns the right set of titles on the second page.